extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -319,17 +321,43 @@ tags: {  # <amp-story-bookend>
     }
   }
 }
+tags: {  # amp-story-consent (json)
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-story-consent extension .json script"
+  requires_extension: "amp-consent"
+  requires_extension: "amp-story"
+  mandatory_parent: "AMP-STORY-CONSENT"
+  unique: true
+  satisfies: "amp-story-consent extension .json script"
+  attrs: { name: "nonce" }
+  attrs: {
+    name: "type"
+    mandatory: true
+    value_casei: "application/json"
+    dispatch_key: NAME_VALUE_PARENT_DISPATCH
+  }
+  cdata: {
+    blacklisted_cdata_regex: {
+      regex: "<!--"
+      error_message: "html comments"
+    }
+  }
+}
 tags: {  # <amp-story-consent>
   html_format: AMP
   tag_name: "AMP-STORY-CONSENT"
   requires_extension: "amp-consent"
+  requires_extension: "amp-story"
   mandatory_parent: "AMP-CONSENT"
+  requires: "amp-story-consent extension .json script"
```

Comment:
what does this do?

Reply:
this requires the json script to be present within amp-story-consent.

Reply:
I see. Thanks!